src/components/vm/consoles/VncConsole.jsx

```diff
@@ -55,7 +55,7 @@ export const VncConsole = ({
     repeaterID = '',
     vncLogging = 'warn',
     consoleContainerId,
-    onDisconnected = () => {},
+    onDisconnected = (event) => {},
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2038-7e41038d-20250806-115005-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)